documentation/src/main/asciidoc/user_guide/hibernate-ogm.adoc

```diff
@@ -0,0 +1,47 @@
+==  Hibernate OGM
+Hibernate can perform CRUD operations directly on an Infinispan cluster.
+Hibernate OGM is an extension of the popular Hibernate ORM project which makes
+the Hibernate API suited to interact with NoSQL databases such as Infinispan.
+
+From the perspective of the Hibernate user, it might be interesting to replace
+the RDBMS with Infinispan to benefit from the better scalability and higher
+performance on CRUD operations; the drawback is that Infinispan - not being a
+relational database - can not run complex relational queries.
+
+From the perspective of the Infinispan user, using Hibernate OGM can simplify
+the learning curve as the JPA API is quite simple to use and already well known
+by many developers.
+
+But Hibernate OGM brings many more benefits, not least experience with using
+Infinispan and transparently addressing several problems you might not even
+have thought about.
+
+For example it does not "serialize" your Java POJOs into Infinispan, as that
+means you'd not be able to de-serialize your data when you make any changes
+to your model.
+Would you like to have to wipe your database for any change to your app?
+
+In the best of cases that wouldn't be very efficient; in some worse scenarios
+your team might not have thought of this and you get stuck into a complex
+migration on your live data.
+Hibernate OGM - just like when using it on a relational database via Hibernate
+ORM - will deconstruct your objects into stable primitives and store these into
+Infinispan instead.
+Being able to "map" new domain objects by simply adding a couple of annotations
+is going to make you far more productive than re-inventing such deconstruction
+techniques, or figuring out how to best store object graphs and relations
+into Infinispan.
+
+Finally, using Hibernate OGM will allow you to use all existing framework
+integration points, such as injecting an `EntityManager` as usual: it's not
+yet another tool but it's the real Hibernate, so inheriting all well known
+integrations: this will work in JavaEE, Spring, Grails, Jhipster, ... and all
```

Comment:
Do you feel confident about giving some projects as examples? We haven't tested it